src/cloudWatchLogs/commands/copyLogStreamName.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import { parseCloudWatchLogsUri } from '../cloudWatchLogsUtils'
+
+export async function copyLogStreamName(uri?: vscode.Uri): Promise<void> {
+    if (!uri) {
+        uri = vscode.window.activeTextEditor?.document.uri
+        if (!uri) {
+            vscode.window.showErrorMessage(
+                localize('aws.cloudWatchLogs.invalidEditor', 'Current editor is not a valid Cloudwatch Logs editor.')
+            )
+            return
+        }
+    }
+    try {
+        const parsedUri = parseCloudWatchLogsUri(uri)
+        await vscode.env.clipboard.writeText(parsedUri.streamName)
+    } catch (e) {
+        vscode.window.showErrorMessage(
+            localize('aws.cloudWatchLogs.invalidEditor', 'Current editor is not a valid Cloudwatch Logs editor.')
```

Comment:
"editor" is vscode extension jargon. And "current" is usually a strange thing to mention in a message, since it is implied: if we were giving a message about a non-current thing, _that_ would be worth mentioning.

We can say something more intuitive:

    localize(..., 'Not a Cloudwatch Log steam: %s', vscode.window.activeTextEditor?.document.name)

- variable part goes at the end
- `document.name` is pseudocode for "whatever the 'friendly name' of the document is"